packages/playground/testUtils.ts

```diff
@@ -63,7 +63,7 @@ function rgbToHex(rgb: string): string {
   }
 }
 
-const timeout = (n: number) => new Promise((r) => setTimeout(r, n))
+export const timeout = (n: number) => new Promise((r) => setTimeout(r, n))
```

Comment:
This is no longer needed
```suggestion
const timeout = (n: number) => new Promise((r) => setTimeout(r, n))
```